packages/bot/src/scratch/blocks/Binary/Tools/Misc./balance.js

```diff
@@ -4,15 +4,13 @@ import { translate } from '../../../../../utils/lang/i18n';
 Blockly.Blocks.balance = {
     init() {
         this.jsonInit(this.definition());
+        this.setOutputShape(Blockly.OUTPUT_SHAPE_ROUND);
```

Comment:
This is redundant as the shape is already set in the block definition.

```suggestion
```

Reply:
done